docs/x509/certificate-transparency.rst

```diff
@@ -76,4 +102,39 @@ issued.
         For SCTs corresponding to pre-certificates.
 
 
+.. class:: HashAlgorithm
+
+    .. versionadded:: 38.0
+
+    An enumeration for SignedCertificateTimestamp hash algorithms.
+
+    .. attribute:: NONE
+
+    .. attribute:: MD5
+
+    .. attribute:: SHA1
+
+    .. attribute:: SHA224
+
+    .. attribute:: SHA256
+
+    .. attribute:: SHA384
+
+    .. attribute:: SHA512
+
+
+.. class:: SignatureAlgorithm
+
+    .. versionadded:: 38.0
+
+    An enumeration for SignedCertificateTimestamp signature algorithms.
+
+    .. attribute:: ANONYMOUS
+
+    .. attribute:: RSA
+
+    .. attribute:: DSA
+
+    .. attribute:: ECDSA
```

Comment:
Flagging for review: I could add some language to each of these attributes, but I wasn't sure how advantageous it would be (since it's mostly be "the `{algorithm}` algorithm" pasted over and over).

Reply:
Sounds good, will do now.

Reply:
96eea780..52d0d97d